heir_py/pipeline_test.py

```diff
@@ -0,0 +1,50 @@
+from heir_py import openfhe_config
+from heir_py.pipeline import run_compiler
+
+from absl.testing import absltest  # fmt: skip
+
+
+class PipelineTest(absltest.TestCase):
+    def test_simple_arithmetic(self):
+        def foo(a, b):
+            return a * a - b * b
+
+        _heir_foo = run_compiler(foo, openfhe_config=openfhe_config.from_os_env())
+
+        cc = _heir_foo.foo__generate_crypto_context()
+        kp = cc.KeyGen()
+        _heir_foo.foo__configure_crypto_context(cc, kp.secretKey)
+        arg0_enc = _heir_foo.foo__encrypt__arg0(cc, 7, kp.publicKey)
+        arg1_enc = _heir_foo.foo__encrypt__arg1(cc, 8, kp.publicKey)
+        res_enc = _heir_foo.foo(cc, arg0_enc, arg1_enc)
+        res = _heir_foo.foo__decrypt__result0(cc, res_enc, kp.secretKey)
+
+        self.assertEqual(-15, res)
+
+    # def test_branch(self):
```

Comment:
could you add a quick comment here saying that heir-opt's pipelines don't support cf dialect (and that's why this test is commented out, not because of part of the python compiler pipeline) just in case someone is checking out the frontend tests?

Reply:
I just removed the test, I will add it back in a future PR